edsdme/blocks/announcement-date/announcement-date.js

```diff
@@ -0,0 +1,37 @@
+import { formatDate, getLocale } from '../../scripts/utils.js';
+import { createTag } from '../utils/utils.js';
+
+const CARD_METADATA_PROPERTY_CREATED = 'created';
+
+export default async function init(el) {
+  performance.mark('announcement-date:start');
+
+  let createdDateValue;
+  const cardMetadataEl = document.querySelector('.card-metadata');
+
+  if (cardMetadataEl) {
+    [...cardMetadataEl.children].forEach((row) => {
+      const firstColumn = row.children[0];
+      if (firstColumn && firstColumn.innerText.trim().toLowerCase() === CARD_METADATA_PROPERTY_CREATED) {
+        createdDateValue = row.children[1]?.innerText.trim();
+      }
+    });
+  }
+
+  const createdDate = new Date(createdDateValue);
+  const locale = getLocale();
+
+  if (!isNaN(createdDate)) {
+    el.innerHTML = '';
+    el.className = `announcement-date-wrapper content ${el.className}`;
+    el.classList.remove('announcement-date');
+    const month = String(createdDate.getMonth() + 1).padStart(2, '0');
```

Comment:
@richardhand-fortedigital This is great, but if the date must me authored in the same format, we can use shorter code:
`    const splitDate = createdDateValue.split('/');
    const datetime = `${splitDate[2]}-${splitDate[0]}-${splitDate[1]}`;`

Reply:
@draganatrajkovic do we know that MM/DD/YYYY is the only accepted format in the caas cards? Don't see any documentation on the format https://milo.adobe.com/docs/authoring/send-to-caas

I find using the parsed date object more robust and future proof even if it's a little less succinct.

Reply:
@richardhand-fortedigital From my experience, the authoring format should always be: MM/DD/YYYY, but I also cannot find documentation to support this :)
I agree with you, your solution is safe.